org.eclipse.buildship.core/src/main/java/org/eclipse/buildship/core/internal/workspace/ConnectionAwareLauncherProxy.java

```diff
@@ -44,7 +44,7 @@
  * @author Donat Csikos
  */
 @SuppressWarnings("unchecked")
-final class ConnectionAwareLauncherProxy implements InvocationHandler {
+public final class ConnectionAwareLauncherProxy implements InvocationHandler {
```

Comment:
❓ Why was this changed?

Reply:
It's just a residue from my fiddling with the test coverage. It should not be there.

Reply:
Fixed